server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/UserRoutes.java

```diff
@@ -61,6 +67,44 @@
 import spark.Service;
 
 public class UserRoutes implements Routes {
+    interface UserCondition {
+        boolean satisfyCondition(UserResponse user);
+    }
+
+    static class HasNoMailboxesCondition implements UserCondition {
```

Comment:
Forgive me but are those ever injected or we did just write unexposed useless code???